platform-sdk/docs/base/logging/logging.md

```diff
@@ -0,0 +1,172 @@
+
+   ```java
+   public class MyClass {
+       private static final Logger LOGGER = Loggers.getLogger(MyClass.class);
+   }
+   ```
+
+   This statement creates a logger specifically for `MyClass`, facilitating targeted logging within this class.
+
+### Logging Messages
+
+Once you have set up the logger, you can proceed to log messages. The new API provides a straightforward and flexible way to log information. Here’s how to use it:
+
+1. **Basic Logging**:
+   For a simple log message, use the `info` method with a string argument.
+
+   ```java
+   LOGGER.info("Hello, world!");
+   ```
+
+2. **Logging with Placeholders**:
+   The API supports placeholder syntax for dynamic message composition. Use curly braces `{}` as placeholders and pass the dynamic values as additional arguments.
+
+   ```java
+   LOGGER.info("Hello, {}!", "world");
+   ```
+
+   In this example, the `{}` placeholder in the string will be replaced by `"world"`, resulting in the log message `"Hello, world!"`.
+
+
+---
+
+
+## Custom Logging Configuration
+
+The configuration syntax is designed for clarity and ease of use. It allows setting a global default logging level and specific levels for packages or classes, with an option to introduce filters for markers.
+For practicality, the `logging.properties` file should be set to reload every second, ensuring up-to-date logging configurations at all times.
+
+Regarding the format of the logging configuration, it should be user-friendly and self-explanatory. Here's an improved representation of the syntax for setting logging levels:
+
+```properties
+# Global default logging level
+logging.level = INFO
+
+# Specific logging levels for packages or classes
+logging.level.com.swirlds.common.crypto = DEBUG
+logging.level.com.swirlds.common.crypto.Signature = WARN
+logging.level.com.hashgraph = WARN
+```
+
+This configuration demonstrates how to set a global default logging level (`INFO`). It also illustrates how to specify logging levels for packages and classes. For example, everything under `com.swirlds.common.crypto` is set to `DEBUG`, except for `com.swirlds.common.crypto.Signature`, which is explicitly set to `WARN`. Similarly, all loggers within `com.hashgraph` default to `WARN`. This approach ensures that loggers inherit the most specific level defined in the configuration, providing both flexibility and precision in logging management.
+
+To provide developers with more control and flexibility in logging, we propose introducing filters for markers. This feature would allow developers to focus on log messages associated with specific markers, regardless of the logger's log level. If there's demand for such functionality among developers, the configuration for marker filters could be structured as follows:
+
+```properties
+# Marker filter configuration
+logging.marker.CONFIG = ENABLED
+logging.marker.CRYPTO = DISABLED
+logging.marker.OTHER = DEFAULT
+```
```

Comment:
I'm not convinced about the style in which it is written...
I think that if the feature is available, it should explain what filters are, how to configure, what they provide after configuring, and leave out phrases like "we propose introducing...", " This feature would allow ..."; "If there's demand for such functionality among developers ..."
And if is not available, I wouldn't make this document the space for discussion.